agents/gce/fence_gce.py

```diff
@@ -9,13 +9,16 @@
 #
 
 import atexit
+import httplib2
 import logging
 import json
 import re
 import os
 import socket
+import socks
```

Comment:
You can move this down to the `try/except: pass` below, and you should be able to remove PySocks from `.travis.yml`.

Reply:
I moved the import and updated the travis.yml file